cmd/clusterctl/configuration/ibmcloud/clouds.yaml.template

```diff
@@ -1,3 +1,2 @@
-clouds:
-  username: <username>
-  toke: <toke>
+userName: "your-name"
```

Comment:
I think we should keep the format as follows to enable future extension.

```
clouds:
  ibmcloud:
    auth:
      username: "test@test.com"
      password: "here'syourpassword"
```

Reply:
I will implement it when the whole creation path succeed.